### ISSUE TITLE
Raspberry Pi 4 Model  Raspberry Pi OS (64 bit )

### ISSUE THREAD

**Issue by marellanrao** [raspberrypi/Raspberry-Pi-OS-64bit#16]
I have tried Raspberry Pi OS (64bit) for Raspberry Pi 4 Model B through SD card.Its ok.But after copying to WD SSD(240GB) os is loading but internet issue is coming.
Anybody can help me to solve the issue.

**Comment by lurch** [comment]
I'm afraid you'll need to provide **much** more info than that - from your brief description it's impossible to tell what your problem is, let alone what might be causing it...

**Comment by mehulboricha** [comment]
I did encounter the internet issue. In my case, it was a strange DNS problem. However, this fixed it:

https://raspberrypi.stackexchange.com/questions/64556/problem-with-dns

**Comment by fishd72** [comment]
Can confirm I'm running the 64-bit beta on a Raspberry Pi 4 and have used wifi and ethernet, both from an SD card and booting from a USB connected SSD. No issues found with internet/network connectivity so far.

**Comment by marellanrao** [comment]
First of all I have installed Raspberry Pi OS(64 bit) on 32GB memory card
and used in my Raspberry Pi 4 Model B(4GB).
The software runs smoothly.
After that I have copied the os from memory card to WD SSD(240 GB).I have
removed the memory card and and started the Pi 4, it boosted without Any
problem.
But Ethernet is not working.
Is there any modifications to be done to run Internet.
I am amature in software.

Narasimha Rao Marella
